### ISSUE TITLE
Bug: sort-union-types broken

### ISSUE THREAD

**Issue by k-teplyakov** [azat-io/eslint-plugin-perfectionist#33]
### Describe the bug

sort-union-types considers an inline comment as part of a union type and breaks it by sorting it

### Code example

This case is ok
```diff
-    accountTypeIds?: (1 | 2 | 3 | 4 | 5 | 100)[]; // Specific Account type Ids to filter accounts. Example: 2
+    accountTypeIds?: (100 | 5 | 4 | 3 | 2 | 1)[]; // Specific Account type Ids to filter accounts. Example: 2
```

And here is the problem
```diff
-    accountTypeId: 1 | 2 | 3 | 4 | 5 | 100; // Account type ID. Example: 3
+    accountTypeId: 100 | // Account type ID. Example: 3 5 | 4 | 3 | 2 | 1;
```

### ESLint version

8.21.0

### ESLint Plugin Perfectionist version

1.5.0

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Thank you for issue. I think I can look into this more on Friday.

**Comment by azat-io** [comment]
Fixed in 95560abdad35184a3d88598fccc7e45956804a4b.

Release is coming soon.

**Comment by azat-io** [comment]
Released in [v2.0.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.0)
